issue4/navigation_test.js

```diff
@@ -0,0 +1,61 @@
+var file_path = process.env.FILE_PATH
+var webdriver = require('selenium-webdriver');
+var assert = require('chai').assert;
+var { Builder, By } = require("selenium-webdriver");
+
+
+
+describe("Testing Navigation on HtmlForms", function() {
+    this.timeout(3 * 1000 * 60);
+    var driver = new webdriver.Builder().forBrowser('chrome').build()
+
+    before(function setupWebdriver(done) {
+        var file_url = `file:///Users/user/Downloads/Dufuna-CodeCamp20/submissions/oyetejuOloyede/htmlForms/contact.html`;
```

Comment:
The `file_url` value shouldn't be an hardcoded value. You might want to check other test cases to see how this was implemented?